### ISSUE TITLE
Name beginning with comma doesn't read aloud in People pagination

### ISSUE THREAD

**Issue by erinchambers** [CDRH/earlywashingtondc#258]
OSCYS has one record that "begins with a comma" (according to the People page)—", Dinah."

I don't think this is an error persay, but the result is that in the "Browse By Letter" buttons, the screen reader reads nothing when it gets to the comma.

> Link, All, list, 26 items
> Link, 1 items
> Link, a, 186 items

It'd be best to have _something_ for the screen reader to announce—"No Last Name," or even just the word "Comma"—but I'm not sure how to do that given our current `people_helper.rb` setup. Is there a way to say "If `,`, then ..." ?

People page: https://cdrhrailsdev.unl.edu/oscys/people
"1 People start with ," page: https://cdrhrailsdev.unl.edu/oscys/people/all?letter=%2C

**Comment by erinchambers** [comment]
I started putzing around with this file in the `people_pagination` branch when I was adding singular/plural support ("person" vs. "people")
